dataforest/core/ProcessRun.py

```diff
@@ -85,7 +85,13 @@ def process_path_map(self) -> Dict[str, Path]:
 
     @property
     def process_plot_map(self) -> Dict[str, Path]:
-        return {file_alias: self.plots_path / self._plot_lookup[file_alias] for file_alias in self._plot_lookup}
+        plot_map_dict = {}
```

Comment:
Docstring would be helpful

Reply:
Also looks like you broke it into several understandable steps